sql/postgresql/README.md

```diff
@@ -25,7 +25,73 @@ createdb my_pap_db
 psql --dbname=my_pap_db --file="<pingauthorize-contrib-root>/sql/postgresql/init-policy-database.sql" >/dev/null
 ```
 You should now have an initialized policy database. For instructions on configuring the Policy Editor to connect to
-your PostgreSQL instance, see "[Installing the PingAuthorize Policy Editor manually](https://docs.pingidentity.com/csh?Product=paz-latest&context=paz_install_pe_noninteractive)".
+your PostgreSQL instance, see [Installing the PingAuthorize Policy Editor manually](https://docs.pingidentity.com/csh?Product=paz-latest&context=paz_install_pe_noninteractive).
+
+
+### Upgrading an existing policy database to a newer schema
+1. Verify the current version of your policy database using one of the following methods.
+
+**Optional** - Access `my_pap_db` and query the `TAG` column of `my_pap_db.databasechangelog`
+
+```
+psql -U postgres -d my_pap_db
+```
+
+```
+SELECT TAG FROM DATABASECHANGELOG WHERE TAG IS NOT NULL ORDER BY TAG DESC LIMIT 1;
+```
+
+**Optional** - Follow the [steps to non-interactively set up](https://docs.pingidentity.com/csh?Product=paz-latest&context=paz_install_pe_noninteractive)
+   the target version of the Policy Editor. If your schema is out of date, you will be prompted to run the intermediate upgrade scripts sequentially.
+
+```
+bin/setup demo \
+  --adminUsername admin \
+  --dbConnectionString "jdbc:postgresql://<host>:<postgresql_port>/my_pap_db" \
+  --dbAppUsername "<postgres user>" \
+  --generateSelfSignedCertificate \
+  --decisionPointSharedSecret pingauthorize \
+  --hostname <host> \
+  --port <pap port> \
+  --adminPort <admin port> \
+  --licenseKeyFile <license>
+```
+
+_Setup completed successfully_
+
+_Please start the server by running bin/start-server_
+
+```
+<PingAuthorize-PAP>bin/start-server
+```
+
+_The policy database at 'jdbc:postgresql://localhost:5432/my_pap_db' is older than this version of PingAuthorize (9.1.0.0). Please use the following scripts to upgrade the policy database before running start-server again:_
+
+_https://github.com/pingidentity/pingauthorize-contrib/blob/main/sql/postgresql/9.1-GA.sql_
+
+_https://github.com/pingidentity/pingauthorize-contrib/blob/main/sql/postgresql/9.1-EA.sql_
```

Comment:
I think the intent was for the URLs to be printed in the order that they should be applied, but here it looks like they are in the reverse order (I would expect the "EA" script to be applied before the "GA" script).

This could be a defect in the `setup` tool. Would you check that and create an internal ticket to address it if that's the case?  As a strawman, I could see it being very helpful to order them correctly and also prefix them with an index to remove any ambiguity.

Reply:
It appears to be a defect in `setup` as this is the order in which they were printed out. Thanks for catching that.

I will follow up and make sure that is resolved.